src/wwwroot/locales/es-translation.js

```diff
@@ -273,6 +276,8 @@ window['relay-translation-es'] = {
   "dkim_configuration_help_spf_subtitle": "Incluye el SPF de {{ companyName }} para evitar la falsificación de direcciones de tu remitente. A continuación puedes encontrar un ejemplo de cómo debería quedar el registro para utilizar el SPF de {{ companyName }}. Es recomendable que el registro sea de tipo TXT y no SPF.",
   "dkim_configuration_help_spf_subtitle_link": "Aquí puedes encontrar ayuda para configurar el SPF.",
   "dkim_configuration_help_spf_subtitle_link_href": "es/como-configurar-tu-dkim-paso-a-paso#spf",
+  "resubscribe_title": "Resubscribir dirección de correo",
+  "resubscribe_help_field": "Aquí puede eliminar una dirección de correo electrónico del estado 'Dropped'",
```

Comment:
What do you think abound this change to be equivalent to English text?
```suggestion
  "resubscribe_help_field": "Aquí puede remover una dirección de correo electrónico del estado 'Dropped'",
```